src/docs/release/breaking-changes/container-color.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Container with color optimization
-description: A container with a color and no other background decoration no
-longer builds the same child widgets.
+description: A container with a color and no other background decoration no longer builds the same child widgets.
```

Comment:
This line is what broke the build, because it had a linefeed in it. A description shouldn't have a linefeed.